packages/cli/src/commands/tile/watch.ts

```diff
@@ -0,0 +1,30 @@
+import chalk from 'chalk'
+import { TileDocument } from '@ceramicnetwork/stream-tile'
+
+import { Command } from '../../command'
+import type { CommandFlags } from '../../command'
+
+export default class Watch extends Command<CommandFlags, { streamId: string }> {
+  static description = 'monitor stream for any updates'
+
+  static args = [
+    {
+      name: 'streamId',
+      required: true,
+      description: 'ID of the stream',
+    },
+  ]
+
+  async run(): Promise<void> {
+    this.spinner.start('Loading stream...')
+    try {
+      const doc = await TileDocument.load(this.ceramic, this.args.streamId)
```

Comment:
does this actually work? In the Ceramic CLI it runs once, prints the most recent commit, then immediately shuts down.  The actual desired behavior would probably be for the CLI to keep running until the user manually sends it a SIGTERM, and continue to output new commits as they come in over time.

We should probably either fix this, or just remove it for now.

Reply:
I'd expect this is how it works as the CLI process should stay on as long as there's an active timer, but I haven't checked, cc @Sterahi

Reply:
See above, this was logic stolen entirely from the Ceramic CLI & if it isn't working there then I"m going to be removing it today.